src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -431,6 +433,13 @@ private void WriteBootConfig(string entryAssemblyName)
                 result.extensions[key] = config;
             }
         }
+        Profilers ??= Array.Empty<string>();
+        var browserProfiler = Profilers.FirstOrDefault(p => p.StartsWith("browser:"));
+        if (browserProfiler != null)
+        {
+            result.environmentVariables ??= new();
+            result.environmentVariables["DOTNET_WasmPerfInstrumentation"] = browserProfiler.Substring("browser:".Length);
+        }
```

Comment:
You can move this code into the `helper` and share it

Reply:
nope, let's delete the other one

Reply:
Perhaps we can make the whole json generation "helper" ? In another PR